src/app.tsx

```diff
@@ -1,6 +1,19 @@
 import React from "react";
-import { OpencertsGallery } from "./opencerts";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { Home } from "./pages/home";
+import { About } from "./pages/about";
 
 export const App: React.FunctionComponent = () => {
-  return <OpencertsGallery />;
+  return (
+    <Router>
```

Comment:
You can put navbar and footer document here so that it will be displayed on every route

Reply:
yes ok